package.json

```diff
@@ -4,14 +4,14 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "node server.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
     "express": "^4.19.2",
-    "highcharts": "^11.4.4",
-    "highcharts-export-server": "^3.1.1"
```

Comment:
I needed to remove this since it was trying to install puppeteer. Puppeteer is known to be difficult to make work well inside of containers. Happy to revisit this decision once we see a clear need.